### ISSUE TITLE
kebab case in #![allow(..)] is not reported as syntax error

### ISSUE THREAD

**Issue by panstromek** [intellij-rust/intellij-rust#5925]
This code doesn't compile but the plugin doesn't report it.

```rust
#![allow(non-upper-case-globals)]
```

Lint name is copypasted from the docs, where it is written with kebab case, but it should be written in snake case in `allow` attribute.

I know the plugin doens't report many errors, but from what I understand, this doesn't even parse (it seems like that from the compiler error), so I guess it would make sense for the plugin to catch it...
